app/templates/main/error.hbs

```diff
@@ -1,7 +0,0 @@
-{{#if slug}}
```

Comment:
Also, I should mention that I wanted to use the "tears" version of the logo, but we didn't have an `.svg` so it didn't look as crisp. @lislis do you happen to know if we have that?